sd-local.go

```diff
@@ -1,7 +1,19 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"github.com/screwdriver-cd/sd-local/config"
+	"log"
+	"os"
+	"path"
+)
 
 func main() {
-	fmt.Println("hello world")
+	path := path.Join(os.Getenv("HOME"), ".sdlocal")
```

Comment:
Is it better to use https://github.com/mitchellh/go-homedir to get Home directory?

Reply:
thx
fixed